projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "event_queue.h"
+#include "status.h"
+
+typedef enum { LIGHTS_BOARD_FRONT, LIGHTS_BOARD_REAR } BoardType;
```

Comment:
The second enumeration value should be followed by a comma, the first should be followed by `= 0` and you should also have `NUM_LIGHTS_BOARD_TYPES` see @karlding's comment RE: style of enums. (You should always check that this value in in range to prevent out of bounds behaviors).

Reply:
whoops forgot to do that here. fixed